templates/vue/src/components/modals/NodeModal/forms/AppearanceForm.vue

```diff
@@ -95,6 +95,15 @@
                 Hide media button
               </b-form-checkbox>
             </b-form-group>
+            <b-form-group>
+              <b-form-checkbox
+                v-show="hasPageMultiContentParent"
```

Comment:
Why not use `v-if` here? `v-show only stylistically hides the element which is generally not what we want.